superset-frontend/src/visualizations/presets/MainPreset.js

```diff
@@ -91,6 +92,7 @@ export default class MainPreset extends Preset {
         new EventFlowChartPlugin().configure({ key: 'event_flow' }),
         new FilterBoxChartPlugin().configure({ key: 'filter_box' }),
         new EchartsGraphChartPlugin().configure({ key: 'graph_chart' }),
+        new EchartsGaugeChartPlugin().configure({ key: 'gauge_chart' }),
```

Comment:
looks like we're trying to keep this in abc order, so let's put this before the `graph_chart`

Reply:
This chart type should probably be renamed to "gauge".